src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,103 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static final int DEFAULT_INDEX = 10;
+    private Object[] array;
+    private int size;
+
+    public ArrayList() {
+        array = new Object[DEFAULT_INDEX];
+    }
+
+    public void exception() {
+        throw new ArrayListIndexOutOfBoundsException("Index out of bounds for length");
+    }
```

Comment:
It is better to use verbs in method naming, as long as method does something.

Reply:
yes, we should use verbs, and I don't see the reason why we need this method
also if we use it, it should be `private`